### PR TITLE
docs(readme): Fix banner url to work on Pypi and change links to /stable

### PR DESCRIPTION
**Pull request by DenverCoder1**
## Summary

Currently Pypi has a broken image since the url used for the banner is relative.

image

This changes the link to an absolute path so it should work next time Pypi is updated.

I also added a link to the examples folder on the words "examples directory".

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

The links have been tested.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ Quick Example
     bot.run('token')
 
 
-You can find more examples in the examples directory.
+You can find more examples in the `examples directory <https://github.com/nextcord/nextcord/blob/master/examples/>`_.
```
Comment (TAG-Epic) [inline README.rst:113]: Hmmm... on pypi I would want it to redirect to stable... but on github I would want it to redirect it to master... not quite sure how I would be able to do something like that

Comment (DenverCoder1) [reply README.rst:113]: Technically, it could do a a replace in the `setup.py`, but it would be cleaner to leave it unchanged

Comment (DenverCoder1) [reply README.rst:113]: For example

```py
readme = f.read().replace("/master/", f"/{version}/")
```

Comment (TAG-Epic) [reply README.rst:113]: Yeah that's not ideal... Hmmm

Comment (TAG-Epic) [reply README.rst:113]: Maybe make stable the default?

**Review by MaskDuck — APPROVED** [review]
gud ig

File: README.rst (hunk 1 above)
Comment (MaskDuck) [reply README.rst]: maybe not cuz it affect pool request

Comment (MaskDuck) [reply README.rst]: *pull

Comment (MaskDuck) [reply README.rst]: or make a pypi readme file and a normal readme file ig

**Pull request merged**